client/styles/layout/_fullscreen.scss

```diff
@@ -3,10 +3,29 @@
   width: 100%;
   height: 100%;
   flex-flow: column;
+  @include themify() {
+    background-color: getThemifyVariable('background-color');
+  }
+}
+
+.fullscreen-preview__header {
+  text-align: center;
+  font-size: 16px;
```

Comment:
could you use `rem`s instead of pixel values?

Reply:
Oh right, actually these classes were not even used anymore, so I'll just remove them